### DISCUSSION TITLE
Support for target attribute on navbar links

### DISCUSSION THREAD

**Discussion by ajmeese7** [upptime/upptime#613] [Ideas] [answered]
My instance of Upptime Monitor is displayed in an iFrame on one of my websites, so clicking the `GitHub` link breaks due to their security policy. It would be great if I could easily set `target: _blank` or similar directly in my `.uptimerc.yml` file.

I'd be happy to try and take a crack at a PR if necessary :)

**Comment by AnandChowdhary** [comment, marked as answer]
Sounds like a super useful idea, would be happy to accept a PR!

**Reply by AnandChowdhary** [reply]
Adding a line here for target should work: https://github.com/upptime/status-page/blob/54f5f60938f2c6ccdeabba224a540c0de9e8adf9/src/components/Nav.svelte#L28
